src/main/thrift/parquet.thrift

```diff
@@ -369,7 +369,7 @@ enum PageType {
   DATA_PAGE_V2 = 3;
 }
 
-enum SortOrder {
+enum BoundayOrder {
```

Comment:
Should be "BoundaryOrder". Missing the "r".

Reply:
Done